src/components/GiftInput.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Container = styled.div`
+    padding: 30px;
+`
+
+const Input = styled.input`
+    width: 300px;
+    border-radius: 5px;
+    padding: 10px;
+    border: 2px solid #AC47D3;
+    font-size: 16px;
+    margin-right: 30px;
+`
+
+const Button = styled.input`
+    background-color: #AC47D3;
```

Comment:
Seems like we are reusing this colour in a few places, perhaps we could create a file called `constants.js` that contains a list of exportable values. The format for now would be something as simple as a file with this in it
```
export const awesomePurple = '#AC47D3'
```

Then we could `import` it into this file like this
```
import { awesomePurple } from '../constants.js'
```

Then your background colour could be `background-color: ${awesomePurple};`
💜💜💜💜